### PR TITLE
RAB-233: Remove LastOperationsWidget legacy code

### PR DESCRIPTION
**Pull request by ValentinMumble**
🔥

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/ImportExportBundle/DependencyInjection/PimImportExportExtension.php [hunk 1]
```diff
@@ -39,7 +39,6 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('step.yml');
```
Comment (ghoussard) [inline src/Akeneo/Platform/Bundle/ImportExportBundle/DependencyInjection/PimImportExportExtension.php:39]: We can't trash all datagrid related files ? This widget is the last component which use datagrid (for import/export) ?

**Review by ghoussard — APPROVED** [review]
I didn't see any new backbone file or form extension that mount the widget from Job, you forgot it ?

**Review by axelhms — APPROVED** [review]

**Comment by ValentinMumble** [comment]
> I didn't see any new backbone file or form extension that mount the widget from Job, you forgot it ?

The `LastOperationsWidget` is not mounted via a legacy form extension but by the `@akeneo-pim-community/activity` microfrontend that I updated here https://github.com/akeneo/pim-community-dev/pull/17994/files#diff-c6f8675bf394e9fe0343cdb1454bc602c06de17a4f0623f32470f1f0ab0ffa35

File: src/Akeneo/Platform/Bundle/ImportExportBundle/DependencyInjection/PimImportExportExtension.php (hunk 1 above)
Comment (ValentinMumble) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/DependencyInjection/PimImportExportExtension.php:39]: I believe the last datagrids in ImportExportBundle are for the main list of imports and exports

**Pull request merged**